libs/libcommon/src/libcommon/simple_cache.py

```diff
@@ -473,8 +473,15 @@ def get_valid_datasets(kind: str) -> Set[str]:
     return set(CachedResponseDocument.objects(kind=kind, http_status=HTTPStatus.OK).distinct("dataset"))
 
 
-def is_valid_for_kinds(dataset: str, kinds: List[str]) -> bool:
-    return CachedResponseDocument.objects(dataset=dataset, kind__in=kinds, http_status=HTTPStatus.OK).count() > 0
+def is_valid_for_kinds(
```

Comment:
I think I didn't chose the function name well: it would be better to know that only one response needs to be a success (ie: it's an OR, not and AND).

something like:

```suggestion
def has_any_successful_response(
```

feel free to ignore or change the name to something clearer

Reply:
Sure! Done in https://github.com/huggingface/datasets-server/pull/1618/commits/6f0bda0f19285519098a66d5eb90b82cc545f572